tslint.json

```diff
@@ -64,8 +64,7 @@
       "check-module",
       "check-operator",
       "check-separator",
-      "check-type",
-      "check-typecast"
```

Comment:
I would prefer to ignore prettier occasionally rather than removing tslint rules. Tslint is more important than prettier IMO.
To ignore prettier, add `// prettier-ignore` before the line to ignore (with eventually a comment explaining why)

Reply:
Let's leave as-is for now. Besides, that should be in its own PR. We can revisit this later.